metrics-scraper/monitoring.go

```diff
@@ -159,33 +226,48 @@ func nodeVersion(ctx context.Context, client *rpc.Client, moniker, networkID str
 	return nil
```

Comment:
```suggestion
	return
```

Reply:
addressed in: 94e0597d570fbb1a694bc047ed7b89d8570a1289